### PR TITLE
Enable metadata validation library on staging

### PR DESCRIPTION
**Pull request by vimleshtna**

### CODE REVIEW

File: root_locals.tf [hunk 1]
```diff
@@ -124,6 +124,6 @@ locals {
 
   //feature access blocks
   block_assign_file_references = local.environment == "intg" ? false : true
-  block_validation_library     = local.environment == "intg" ? false : true
+  block_validation_library     = (local.environment == "intg" || local.environment == "staging") ? false : true
```
Comment (TomJKing) [inline root_locals.tf:127]: Not that its that important but can change this to: `block_validation_library = local.environment == "prod" ? true : false`

Just a little bit simpler.

**Review by TomJKing — COMMENTED** [review]
Looks good. Just a minor comment. Thanks

File: root_locals.tf (hunk 1 above)
Comment (vimleshtna) [reply root_locals.tf:127]: Yeah, thanks.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**